### DISCUSSION TITLE
Linguist doesn't work locally for a single HTML file that needs to be excluded

### DISCUSSION THREAD

**Discussion by zkkv** [github-linguist/linguist#6913] [General]
## Describe the bug

I have two relevant files in my project root: `foo.html` and `.gitattributes`. I want to exclude `foo.html` from the linguist's calculation. In `.gitattributes` I tried doing the following:
```
foo.html -linguist-documentation  # As in the documentation
```

```
foo.html linguist-documentation  # Different from the documentation
```

In the latter case, when I run `git check-attr linguist-documentation foo.html` it says the attribute is set to true. In neither cases, however, is HTML excluded from linguist's report, i.e. I get the same result with or without the addition of `.gitattributes` file.

### Expected behaviour

HTML should be excluded from the report.

### Related discussion

None that I could find.

### Additional notes

None.

**Comment by lildude** [comment]
The override works perfectly:

```console
➜ git init test
Initialized empty Git repository in /Users/lildude/Downloads/trash/test/.git/
➜ cd test
➜ echo "<html>" > foo.html
➜ git add foo.html
➜ git commit -m 'Add foo'
[main (root-commit) a6c2adb] Add foo
 1 file changed, 1 insertion(+)
 create mode 100644 foo.html

➜ github-linguist .
100.00% 7          HTML
➜ echo "foo.html linguist-documentation" > .gitattributes
➜ git add .gitattributes         # ‼️
➜ git commit -m 'Add override'   # ‼️ You probably missed these steps
[main 20b5373] Add override
 1 file changed, 1 insertion(+)
 create mode 100644 .gitattributes
➜ github-linguist .

➜
```

I suspect you missed this part from the docs and highlighted above:

https://github.com/github-linguist/linguist/blob/39fd5e93de98de3434cb3e857c5b1972c418f8c5/docs/overrides.md?plain=1#L11

**Reply by zkkv** [reply]
Thanks, I missed that indeed.

**Reply by zkkv** [reply]
If I may ask, why do you have to write it without a dash? Is that a mistake in the documentation?

**Reply by lildude** [reply]
The dash negates the option. `linguist-documentation` says "mark these files as documentation" `-linguist-documentation` says "don't mark the files as documentation". This is a function of gitattributes, not linguist specifically.
